Dockerfile

```diff
@@ -1,22 +1,22 @@
-FROM alpine:3.11
+FROM alpine:3.12
 
 RUN apk --no-cache add \
     curl \
     git \
     libc6-compat \
     openssh-client \
-    python \
-    py-crcmod \
-    py-pip && \
-    pip install --upgrade pip==18.1
+    py3-pip \
+    py3-wheel \
```

Comment:
I'm not a Python expert. Why would we want `py3-wheel`? The image builds fine without it.

Reply:
The reason to install `wheel` is to avoid this warning that you'll see if you run `pip` and don't have `wheel` installed:
```
Using legacy 'setup.py install' for pyyaml, since package 'wheel' is not installed.
```
For an overview of what wheels are and why they're better than `setup.py`, there's a reasonable overview [here](https://realpython.com/python-wheels/).